bindings/ergo-lib-ios/Sources/ErgoLib/NodeInfo.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import ErgoLibC
+
+class NodeInfo: FromRawPtr {
+    internal var pointer: NodeInfoPtr
+
+    internal init(withRawPointer ptr: NodeInfoPtr) {
+        self.pointer = ptr
+    }
+
+    static func fromRawPtr(ptr: UnsafeRawPointer) -> Self {
+        return NodeInfo(withRawPointer: OpaquePointer(ptr)) as! Self
+    }
```

Comment:
Any benefit to having `fromRawPtr` over the constructor?

Reply:
Oh, it's an implementation of `FromRawPtr` protocol I made for `wrapClosure` - https://github.com/ergoplatform/sigma-rust/blob/92fb1e3a3b7e440734bb08b7b1f1502c8fb646b3/bindings/ergo-lib-ios/Sources/ErgoLib/WrapClosure.swift#L60 to get wrapped Swift response type out of Rust's pointer.